packages/line/src/Line.js

```diff
@@ -107,6 +107,7 @@ const Line = props => {
         enableTouchCrosshair = false,
 
         role = 'img',
+        initialHiddenIds = [],
```

Comment:
Sorry, but I missed 2 things during my initial review:
- It needs to be added to the types as this package hasn't been migrated to TypeScript yet
- This property should also be added to the documentation (https://github.com/plouc/nivo/blob/master/website/src/data/components/line/props.ts)

Reply:
hi @plouc i've added to the documentation, however i'm just not sure what you mean by this needing to be added to the types, which file are you referring to? thank you!

Reply:
Hi @ccatalan6, I'm referring to this file: https://github.com/plouc/nivo/blob/master/packages/line/index.d.ts

Reply:
i've added to the types, let me know if there are concerns @plouc

Reply:
@clcatalan, thank you, the build is failing due to formatting issues, please run `make fmt`.

Reply:
hi @plouc  i've run make fmt, but it seems the build is still failing? i checked out the ci, and im not quite sure how to resolve this
![build failing](https://github.com/user-attachments/assets/0d8cbe3c-0932-4f83-ba25-879fdb425057)

Reply:
@clcatalan, this build was broken before your changes, it's unrelated, all good.